src/services/errors.js

```diff
@@ -18,5 +18,9 @@ module.exports= {
 	TagAlreadyAssignedToPostError : {
 		name: 'TagAlreadyAssignedToPostError',
 		message: 'Tag is already assigned to post '
+	},
+	UnauthorizedError : {
```

Comment:
@beblicarl error should be about the refresh token being invalid

Reply:
yes please

Reply:
yes please